src/NServiceBus.Core/Routing/AssemblyRouteSource.cs

```diff
@@ -20,7 +20,7 @@ public AssemblyRouteSource(Assembly messageAssembly, UnicastRoute route)
         public IEnumerable<RouteTableEntry> GenerateRoutes(Conventions conventions)
         {
             var routes = messageAssembly.GetTypes()
-                .Where(t => conventions.IsMessageType(t))
+                .Where(conventions.IsMessageType)
```

Comment:
and this https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/InitializableEndpoint.cs#L96  ?